### PR TITLE
YouTube Clone

### PR DESCRIPTION
**Pull request by DcRonan**
### HTML & CSS: Embedding Images & Videos: YouTube Clone.

- Created a clone of YouTube.

- This project was completed with HTML & CSS.

- Embedded videos & images.

### CODE REVIEW

**Comment by DcRonan** [comment]
Hi Oscar, 

Thanks for your feedback!

Will go through the steps now but one thing is the live demo you have asked for? I don't understand the GitHack site. Do you mean a link to our webpage as you would add in GitHub pages?

Thanks,

Dan

File: style.css [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+}
+
+#search-bar {
+    margin-top: 15px;
+    width: 40%;
+    padding: 8px;
+    border: gray solid 1px;
+}
+
+#search-button {
```
[the remark this answers is not in the document]
Comment (DcRonan) [reply style.css:42]: I have switched most ID selectors to classes.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    margin: 3px 0 0 20px;
+}
+
+.aside-video-text:hover {
+    text-decoration: underline;
+}
+
+#next-video {
+    cursor: pointer;
+}
```
[the remark this answers is not in the document]
Comment (DcRonan) [reply style.css:197]: I have merged them into one style.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="images/playbutton.png" type="favicon/png">
+    <script src="https://kit.fontawesome.com/867f1dd431.js" crossorigin="anonymous"></script>
+</head>
+
+<body class="container">
+
+    <nav id="nav-bar">
+        <ul>
+            <li style="float: left;">
```
[the remark this answers is not in the document]
Comment (DcRonan) [reply index.html]: Added live demo and finished all steps.

**Comment by krys2fa** [comment]
# Status: Project Approved 🏁 

Hello @DcRonan and @LamiaSristy, 

Nice work with implementing the requested changes. 👍 

Most importantly, congratulations on completing your first project. 👏 

You can merge this branch and move onto the next project. 🎊

Happy coding! 😄 💻

**Pull request merged**